charts/kong/README.md

```diff
@@ -287,6 +287,7 @@ For a complete list of all configuration values you can set in the
 | podDisruptionBudget.maxUnavailable | Represents the minimum number of Pods that can be unavailable (integer or percentage) | `50%`               |
 | podDisruptionBudget.minAvailable   | Represents the number of Pods that must be available (integer or percentage)          |                     |
 | podSecurityPolicy.enabled          | Enable podSecurityPolicy for Kong                                                     | `false`             |
+| priorityClassName                  | Set priority class for Kong                                                           | ""                  |
```

Comment:
Let's make this more specific. Something along the lines of `Set pod scheduling priority class for Kong pods`.